exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/work_in_progress/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,1170 @@
+{
```

Comment:
<p>I didn't understand much in this section. </p><p>"Consider also ..." Why "also"? It's your first sentence in this section.</p><p>What "further processing steps" you are talking about I have no idea.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='82'/>

Reply:
<p>removed it, the majority of spelling errors I generated for the first version of the dataset, for the moment this section is probably obsolete</p>